### DISCUSSION TITLE
How to analyze tree sequence data?

### DISCUSSION THREAD

**Discussion by lakishadavid** [tskit-dev/tsinfer#541] [Q&A] [answered]
I'm not sure how to proceed after getting results from tsinfer.

I created a tsinfer SampleData object using 4 human individuals. 1 population. 59152 variants. The results:
"Sample file created for 8 samples (4 individuals) with 58945 variable sites."

Results from tsinfer on the sample data:
Trees: 14323
Sample Nodes: 8

Is there guidance on how to make sense of the resultant tree sequence data?

**Comment by hyanwong** [comment, marked as answer]
There's a brief section in https://tsinfer.readthedocs.io/en/latest/tutorial.html#analysis

Note that the branch lengths in an inferred tree sequence may not be meaningful, so normal tskit statistics will only work properly in "sites" mode (not "branch" mode - see https://tskit.dev/tskit/docs/stable/stats.html#mode and https://www.genetics.org/content/215/3/779).

If you want to use branch lengths in the trees, you should run `tsdate` on the resulting tree sequence: https://tsdate.readthedocs.io/en/latest/
